### DISCUSSION TITLE
Label engine show no labels :(

### DISCUSSION THREAD

**Discussion by ferrymiddelkoop** [snipe/snipe-it#14239] [Q&A]
I have update Snipe It to version 6.3.0 but i see no labels in Labels menu. Whats here the problem?
![2024-02-09 18_36_30-Window](https://github.com/snipe/snipe-it/assets/45721392/130a5ebb-3c14-4262-8612-6409f0a1a4df)

**Comment by ferrymiddelkoop** [comment]
Nobody answer?

**Reply by marcusmoore** [reply]
@ferrymiddelkoop hi there.

Is there anything showing up in the `laravel.log` file when you visit that page?
